content/applications/hr/fleet/configuration.rst

```diff
@@ -57,109 +57,129 @@ for existing available cars in this field.
 
 .. _fleet/configuration/manufacturers:
 
+Models
+======
+
 Manufacturers
-=============
+-------------
 
 Odoo *Fleet* comes pre-configured with sixty-six commonly used car and bicycle manufacturers in the
 database, along with their logos. To view the pre-loaded manufacturers, go to :menuselection:`Fleet
-app --> Configuration --> Manufacturers`.
+app --> Configuration --> Models: Manufacturers`.
 
 The manufacturers appear in a list view in alphabetical order. Each manufacturer's card lists how
-many specific models are configured for each particular manufacturer. Odoo comes with forty six
-pre-configured :ref:`models <fleet/configuration/models>` from four major auto manufacturers and one
-major bicycle manufacturer: Audi, BMW, Mercedes, Opel (cars), and Eddy Merckx (bicycle).
+many specific models are configured for each particular manufacturer. Odoo comes with forty five
```

Comment:
Does this number change frequently enough that it would be worth it to say "comes with a number of pre-configured models"? or something similar? I know that you include the actual list so any changes would have to be reflected in the doc anyway, but I wonder if it would be easier to not include the actual number. (Totally your call, thought it was worth a suggestion)

Reply:
Honestly, it only changed because they eliminated a bicycle- which I assume is because they are not a typical "fleet" item people use. So I changed it here since it is only cars now. I see what you are saying, but I feel like it won't change too much- it was consistent up till this switch with the bikes.